mslib/msui/mscolab.py

```diff
@@ -75,6 +75,9 @@
 from mslib.utils.config import config_loader, modify_config_file
 
 
+MSCOLAB_TIMEOUT = tuple(config_loader(dataset="MSCOLAB_timeout"))
```

Comment:
Does this actually make a measurable difference?

Reply:
The downside is that this could lead to using an outdated value when the configuration is changed while the application is running. I _think_ this value wouldn't be updated if `MSCOLAB_timeout` is changed in the configuration editor, even with the application restart that it does.

Reply:
the editor has another bug -> new issue coming. 
<img width="199" alt="Bildschirmfoto 2025-03-24 um 20 05 19" src="https://github.com/user-attachments/assets/316f2443-8f68-4b7e-83f6-5f652aaac7a4" />

and there is another problem by that var or on the MBP
![Bildschirmfoto 2025-03-24 um 20 17 03](https://github.com/user-attachments/assets/fbb711a3-8a91-40be-a82a-6890cdd478aa)

That was not expected.

Reply:
Anyway it is more important to have always a timeout there then a short cut module global. It is an easy refactoring.